samples/bluetooth/fast_pair/locator_tag/src/dfu.c

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <app_version.h>
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/bluetooth/uuid.h>
+#include <bluetooth/adv_prov.h>
+
+#include <bluetooth/services/fast_pair/fast_pair.h>
+#include <bluetooth/services/fast_pair/fmdn.h>
+
+#include <zephyr/mgmt/mcumgr/grp/os_mgmt/os_mgmt.h>
+#include <zephyr/mgmt/mcumgr/mgmt/callbacks.h>
+#include <zephyr/mgmt/mcumgr/mgmt/mgmt.h>
+
+#include "app_dfu.h"
+#include "app_factory_reset.h"
+#include "app_fp_adv.h"
+#include "app_ui.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(fp_fmdn, LOG_LEVEL_INF);
+
+/* DFU mode timeout in minutes. */
+#define DFU_MODE_TIMEOUT (1)
```

Comment:
Isn't the timeout a bit too low? Consider using 5 minutes. It takes time to open nRF Connect DM, find the advertising peer, select a DFU package, establish a connection

Reply:
Personally I did not need to have it for longer, but sure, let's increase it.